Source/WebCore/platform/mac/VideoPresentationInterfaceMac.mm

```diff
@@ -45,7 +45,10 @@
 
 #import <pal/cf/CoreMediaSoftLink.h>
 
+#if !defined(SOFTLINK_AVKIT_FRAMEWORK_DEFINED)
+#define SOFTLINK_AVKIT_FRAMEWORK_DEFINED
 SOFTLINK_AVKIT_FRAMEWORK()
+#endif
```

Comment:
added a file in SourcesCoacoa.txt broke unified build and i had to do add these guards. @jyavenard

Reply:
I've had this error before , you can only use SOFTLINK_AVKIT_FRAMEWORK in a file once otherwise you'll get an error about the AVKit symbol being redefined .
you should instead use `SOFT_LINK_FRAMEWORK_FOR_SOURCE` here.

Reply:
the issue is after adding @no-unify i get this error and `PictureInPictureSupport.h` seems like the relevant header. however, i can't find it's implementation

```
ld: warning: ignoring duplicate libraries: '-lobjc', '-lswiftCore'
Undefined symbols for architecture arm64e:
  "WebCore::supportsPictureInPicture()", referenced from:
      WebCore::DocumentPictureInPicture::pictureInPictureEnabled(WebCore::Document&) in UnifiedSource73.o
ld: symbol(s) not found for architecture arm64e
```

Reply:
fixed this by adding an entry to build phase list in xcode